api/evaluation/evaluation.proto

```diff
@@ -0,0 +1,165 @@
+  option (google.api.http) = {
+      post : "/v1/evaluation/evaluate/{cloud_service_id}/{catalog_id}/start"
+    };
+    } 
+
+  // StopEvaluation stops the evaluation for the given target of evaluation
+  rpc StopEvaluation(StopEvaluationRequest) returns (StopEvaluationResponse) {
+  option (google.api.http) = {
+      post : "/v1/evaluation/evaluate/{cloud_service_id}/{catalog_id}/stop"
+      };
+  } 
+
+  // List all evaluation results. Part of the public API, also exposed as REST.
+  rpc ListEvaluationResults(ListEvaluationResultsRequest)
+      returns (ListEvaluationResultsResponse) {
+      option (google.api.http) = {
+          get : "/v1/evaluation/results"
+      };
+  }
+}
+
+message ListEvaluationResultsRequest {
+  // Optional. List only evaluation results for a specific cloud service.
+  optional string filtered_cloud_service_id = 1
+    [ (validate.rules).string.uuid = true ];
+
+  // Optional. List only evaluation results for a specific control id.
+  optional string filtered_control_id = 2
+    [(validate.rules).string.min_len = 1];
+
+  // Optional. List only sub-control evaluation results for a specific control id.
+  optional string filtered_sub_controls = 3
+    [(validate.rules).string.min_len = 1];
+
+  // Optional. Latest results grouped by resource_id and metric_id.
+  optional bool latest_by_resource_id = 4;
+
+  int32 page_size = 10;
+  string page_token = 11;
+  string order_by = 12;
+  bool asc = 13;
+}
+
+message ListEvaluationResultsResponse {
+  repeated EvaluationResult results = 1;
+  string next_page_token = 2;
+}
+
+message StartEvaluationRequest {
+  string cloud_service_id = 1
+    [ (validate.rules).string.uuid = true ];
+  string catalog_id = 2
+    [ (validate.rules).string.min_len = 1 ]; 
+
+  // The interval time in minutes the evaluation executes periodically. The default interval is set to 5 minutes.
+  optional int32 interval = 3
+    [ (validate.rules).int32.gt = 0 ];
+}
+
+message StartEvaluationResponse {
+  bool status = 1; 
+  string status_message = 2; 
+}
+
+message StopEvaluationRequest {
+  string cloud_service_id = 1
+    [ (validate.rules).string.uuid = true ];
+
+  string catalog_id = 2
+    [ (validate.rules).string.min_len = 1 ]; 
+}
+
+message StopEvaluationResponse {}
+ 
+// A evaluation result resource, representing the result after evaluating the cloud service with a specific control
+// cloud_service_id, category_name and catalog_id are necessary to get the corresponding TargetOfEvaluation
+message EvaluationResult {
+  // Evaluation result id
+  string id = 1
+    [(validate.rules).string = {ignore_empty : true, uuid : true}]; 
+
+  // The Cloud Service ID the evaluation belongs to
+  string cloud_service_id = 2
+  [(validate.rules).string = {ignore_empty : true, uuid : true}]; 
+
+  // The control id the evaluation was based on
+  string control_id = 3
+    [(validate.rules).string.min_len = 1];
+  
+  // The category the control belongs to 
+  string category_name = 4
```

Comment:
A better name would probably be:
```suggestion
  string control_category_name = 4
```

Then its clear that it is the category of the control and not a category of the eval result